spec/menu_spec.rb

```diff
@@ -0,0 +1,16 @@
+require './lib/menu.rb'
+
+describe Menu do
+    let(:menu) {described_class.new}
+
+    describe "#initialization" do
+
+    it 'is initialized with a default menu' do
+      expect(subject.menu).to include {}
+    end
+
+    it 'knows the price of each with dish' do
+      expect(subject.price(:Coke)).to eq (5)
+    end
+  end
```

Comment:
Lint/BlockAlignment: `end` at 15, 2 is not aligned with `describe "#initialization" do` at 6, 4.